test/unit/adb-commands-specs.js

```diff
@@ -792,6 +793,13 @@ describe('adb commands', () => {
       await adb.getScreenSize();
       mocks.adb.verify();
     });
+    it('should get device screen density', async () => {
+      mocks.adb.expects("shell")
+          .once().withExactArgs(['wm', 'density'])
+          .returns(screenDensity);
+      await adb.getScreenDensity();
```

Comment:
Plase add the verification of the returned result. Also it would be good to have a test, which verifies `null` output case